cf-agent/verify_files_utils.c

```diff
@@ -3596,13 +3596,13 @@ bool VerifyOwner(EvalContext *ctx, const char *file, const Promise *pp, Attribut
 
     if (attr.perms.groups->next == NULL && attr.perms.groups->gid == CF_UNKNOWN_GROUP)  // Only one non.existent item
     {
-        cfPS(ctx, LOG_LEVEL_ERR, PROMISE_RESULT_FAIL, pp, attr, "Unable to make file belong to an unknown group");
+        cfPS(ctx, LOG_LEVEL_ERR, PROMISE_RESULT_FAIL, pp, attr, "Unable to make file belong to an unknown group '%s'", attr.perms.groups->gid );
         *result = PromiseResultUpdate(*result, PROMISE_RESULT_FAIL);
```

Comment:
Right, so the request was to make this a bit easier to hunt down when it appears in logs. It's not impossible to track down without emitting the user that did not exist, but having the information about the user that was not present gives a better starting point.

I suppose it would be even better if 'file' displayed the full path to the file being promised.

Are you saying there is a better way to emit which user was not found?